core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -532,6 +536,30 @@ private FunctionResolver unix_timestamp() {
     );
   }
 
+  /**
+   * UTC_DATE(). return the current UTC Date in format yyyy-MM-dd
+   */
+  private DefaultFunctionResolver utc_date() {
+    return define(BuiltinFunctionName.UTC_DATE.getName(),
+        impl(DateTimeFunction::exprUtcDate, DATE));
+  }
+
+  /**
+   * UTC_TIME(). return the current UTC Time in format HH:mm:ss
+   */
+  private DefaultFunctionResolver utc_time() {
+    return define(BuiltinFunctionName.UTC_TIME.getName(),
+        impl(DateTimeFunction::exprUtcTime, TIME));
+  }
+
+  /**
+   * UTC_TIMESTAMP(). return the current UTC TimeStamp in format yyyy-MM-dd HH:mm:ss
+   */
+  private DefaultFunctionResolver utc_timestamp() {
+    return define(BuiltinFunctionName.UTC_TIMESTAMP.getName(),
+        impl(DateTimeFunction::exprUtcTimeStamp, TIMESTAMP));
```

Comment:
Return type should be `DATETIME`

Reply:
Fixed in 2ea2397b1e79a33acd12f8468e235b9e5f125901

Reply:
Was it reverted?

Reply:
should be good now